src/image_file.h

```diff
@@ -10,7 +10,7 @@
    Unless required by applicable law or agreed to in writing, software
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
+∫   See the License for the specific language governing permissions and
```

Comment:
what's this?

Reply:
This mistyped character has been deleted.